api/views.py

```diff
@@ -17,7 +18,7 @@ def post(self, request):
         if serializer.is_valid():
             order = serializer.validated_data
 
-            Client(order).create_delivery()
+            create_delivery.delay(order)
 
             return Response(order, status=status.HTTP_201_CREATED)
```

Comment:
I'm wondering if it makes sense to respond with the validated order any more. An empty body is all we need and I never check the response from my terminal, so it's not providing any value.